src/base64.rs

```diff
@@ -1,6 +1,6 @@
 use ::base64::engine::general_purpose::STANDARD;
 use ::base64::engine::Engine;
 
-pub(crate) fn encode<T: AsRef<[u8]>>(input: T) -> String {
```

Comment:
Wasn't it better with `pub(crate)`, making it more obvious that this is an internal only API?

Reply:
Is inside a private module so it is warned about linters